### PR TITLE
356 console errors

### PR DESCRIPTION
**Pull request by ivarni**

### CODE REVIEW

File: styleguide-content/komponenter/knapper.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
--   Starte en prosess. F.eks. «Bli kunde», «Sjekk pris», «Meld skade».
--   Gi en kommando til systemet. F.eks. «Skriv ut», «Last ned», «Betal», «Bestill».
--   Beslutte noe. F.eks. «Kjøp», «Aksepter», «Overfør».
--   Navigere i brukergrensesnittet. F.eks. «Neste», «Fortsett», «Søk».
--   Endre modus eller visning. F.eks. «Redigér» for å gå til redigeringsmodus, «Vis flere» for å ekspandere en liste.
+*   Starte en prosess. F.eks. «Bli kunde», «Sjekk pris», «Meld skade».
+*   Gi en kommando til systemet. F.eks. «Skriv ut», «Last ned», «Betal», «Bestill».
+*   Beslutte noe. F.eks. «Kjøp», «Aksepter», «Overfør».
+*   Navigere i brukergrensesnittet. F.eks. «Neste», «Fortsett», «Søk».
+*   Endre modus eller visning. F.eks. «Redigér» for å gå til redigeringsmodus, «Vis flere» for å ekspandere en liste.
```
Comment (ivarni) [inline styleguide-content/komponenter/knapper.md:7]: I didn't do this...

File: styleguide-content/komponenter/knapper.md [hunk 2]
```diff
@@ -118,17 +120,17 @@ Retningslinjene for hvilke knapper som brukes når er ikke absolutte. Du må ogs
 
 Generelt for alle knapper gjelder:
 
--   Knappen skal tydelig kommuniserer hvilken handling som skjer når brukeren trykker på den, enten ved en beskrivende tekst, et ikon eller begge deler. Tekst på knapper bør være et verb i imperativ form. Unntak er konvensjoner som "Neste".
--   Knappen skal som hovedregel være synlig hele tiden. Knapper som dukker opp etter en rekke handlinger gjør det vanskelig for brukeren å få oversikt.
--   Teksten på knappen skal beskrive hva som skjer når man klikker på knappen. Teksten skal være så kort som mulig og bør ikke bryte over to linjer.
+*   Knappen skal tydelig kommuniserer hvilken handling som skjer når brukeren trykker på den, enten ved en beskrivende tekst, et ikon eller begge deler. Tekst på knapper bør være et verb i imperativ form. Unntak er konvensjoner som "Neste".
+*   Knappen skal som hovedregel være synlig hele tiden. Knapper som dukker opp etter en rekke handlinger gjør det vanskelig for brukeren å få oversikt.
+*   Teksten på knappen skal beskrive hva som skjer når man klikker på knappen. Teksten skal være så kort som mulig og bør ikke bryte over to linjer.
```
Comment (ivarni) [inline styleguide-content/komponenter/knapper.md:125]: ..or this

File: styleguide-content/komponenter/knapper.md [hunk 3]
```diff
@@ -118,17 +120,17 @@ Retningslinjene for hvilke knapper som brukes når er ikke absolutte. Du må ogs
...
+*   Knapper brukes vanligvis til å gjøre handlinger, ikke for å ta valg. For å ta valg, bruk [checkbox](#checkbox) eller [radioknapp](#radioknapper) isteden og forplikt med en knapp.
 
 ### Relaterte komponenter
 
--   [Checkbox](#checkbox)
--   [Radioknapp](#radioknapper)
--   [Lenke](#linktext)
+*   [Checkbox](#checkbox)
+*   [Radioknapp](#radioknapper)
+*   [Lenke](#linktext)
```
Comment (ivarni) [inline styleguide-content/komponenter/knapper.md:136]: ..or any of this

File: styleguide-content/komponenter/knapper.md (hunk 1 above)
Comment (kwltrs) [reply styleguide-content/komponenter/knapper.md:7]: prettier?

Comment (ivarni) [reply styleguide-content/komponenter/knapper.md:7]: Jeg tipper det... 🤷‍♂️

Comment (ivarni) [reply styleguide-content/komponenter/knapper.md:7]: Can confirm...

image

**Review by antidecaf — DISMISSED** [review]

File: packages/ffe-details-list-react/package.json [hunk 4]
```diff
@@ -36,7 +35,8 @@
     "react-dom": "^16.2.0"
   },
   "peerDependencies": {
-    "@sb1/ffe-details-list": "^9.0.2",
+    "@sb1/ffe-grid": "^10.0.1",
+    "@sb1/ffe-grid-react": "^9.0.1",
```
Comment (ivarni) [inline packages/ffe-details-list-react/package.json:41]: The CI build naturally doesn't install peer-deps so I'll try adding the to dev-deps as well in order to fix the build

File: styleguide-content/komponenter/knapper.md (hunk 1 above)
Comment (selbekk) [reply styleguide-content/komponenter/knapper.md:7]: Fint å få det likt på tvers da :+1:

**Review by selbekk — APPROVED** [review]
LGTM!

**Comment by ivarni** [comment]
Coveralls har fortsatt tatt ferie ser det ut som, so this merge is above my paygrade

**Pull request merged**